src/guid.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Unique ID for an element or function
+ * @type {Number}
+ */
+let _guid = 1;
+
+/**
+ * Get a unique auto-incrementing ID by number that has not been returned before.
+ *
+ * @return {number}
+ *         A new unique ID.
+ */
+export function newGUID() {
```

Comment:
This should probably just be `export default function...` which can then be imported more simply.